### ISSUE TITLE
Screen reader does not read prompt in "No Profiling Target" window

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2255]
Repro Steps:
1. Open a solution. 
2. Open the Python Profiling Window
3. Create a new performance Session
4. "Start Profiling" will show this dialog

Screen reader reads the title and then jumps to the buttons without narrating the prompt. This is likely a MAS violation. 

![image](https://cloud.githubusercontent.com/assets/5177491/23766485/d80fb7d2-04b9-11e7-9a48-8b71089fbb63.png)

**Comment by huguesv** [comment]
You need to use CAPS LOCK+LEFT ARROW or CAPS LOCK + RIGHT ARROW to make it read the text.
